cognite/neat/data_model/_constants.py

```diff
@@ -0,0 +1,57 @@
+import re
+import sys
+
+from ._identifiers import NameSpace
+
+if sys.version_info >= (3, 11):
+    from enum import StrEnum
+else:
+    from backports.strenum import StrEnum
+
+
+XML_SCHEMA_NAMESPACE = NameSpace("http://www.w3.org/2001/XMLSchema#")
+
+# This pattern ignores commas inside brackets
+SPLIT_ON_COMMA_PATTERN = re.compile(r",(?![^(]*\))")
+# This pattern ignores equal signs inside brackets
+SPLIT_ON_EQUAL_PATTERN = re.compile(r"=(?![^(]*\))")
+
+
+class EntityTypes(StrEnum):
```

Comment:
Can we manage without this one? I am think we use reflection to pick up all available entity types from the pydantic classes. That means less to maintain